dof/src/dof.rs

```diff
@@ -193,6 +193,11 @@ impl Section {
     pub fn as_bytes(&self) -> Vec<u8> {
         crate::ser::serialize_section(self)
     }
+
+    /// Serialize a section into a JSON representation of the DOF object file section.
+    pub fn to_json(&self) -> String {
+        serde_json::to_string(self).unwrap()
```

Comment:
to_string_pretty?

Reply:
Can do. I had that initially but I figured folks could always use jq, etc